### PR TITLE
Usability tweaks

### PR DESCRIPTION
**Pull request by flurick**
I realize these should probably be separate pull requests, but I'm still not entirely sure how to work this web ui :P

### CODE REVIEW

File: synfig-studio/src/gui/docks/dock_toolbox.cpp [hunk 1]
```diff
@@ -92,8 +92,8 @@ Dock_Toolbox::Dock_Toolbox():
 	Widget_Defaults* widget_defaults(manage(new Widget_Defaults()));
 
 	tool_box_paned = manage(new Gtk::Paned(Gtk::ORIENTATION_VERTICAL));
-	tool_box_paned->pack1(*scrolled_window, Gtk::PACK_EXPAND_WIDGET|Gtk::PACK_SHRINK, 3);
-	tool_box_paned->pack2(*widget_defaults, Gtk::PACK_EXPAND_WIDGET|Gtk::PACK_SHRINK, 3);
+	tool_box_paned->pack1(*scrolled_window, Gtk::PACK_EXPAND_WIDGET|Gtk::PACK_SHRINK, false);
+	tool_box_paned->pack2(*widget_defaults, Gtk::PACK_EXPAND_WIDGET|Gtk::PACK_SHRINK, false);
```
Comment (rodolforg) [inline synfig-studio/src/gui/docks/dock_toolbox.cpp:96]: I think it's wrong :(
The code before and after your changes ;)

File: synfig-studio/src/synfigapp/actions/valuedescexport.cpp [hunk 2]
```diff
@@ -58,7 +58,7 @@ using namespace Action;
 
 ACTION_INIT_NO_GET_LOCAL_NAME(Action::ValueDescExport);
 ACTION_SET_NAME(Action::ValueDescExport,"ValueDescExport");
-ACTION_SET_LOCAL_NAME(Action::ValueDescExport,N_("Export Value"));
+ACTION_SET_LOCAL_NAME(Action::ValueDescExport,N_("Export Value to project library"));
```
Comment (rodolforg) [inline synfig-studio/src/synfigapp/actions/valuedescexport.cpp:61]: I really don't know if it helps :/

File: synfig-studio/src/gui/docks/dock_toolbox.cpp (hunk 1 above)
Comment (rodolforg) [reply synfig-studio/src/gui/docks/dock_toolbox.cpp:96]: OMG now I just realized I only wrote half of may message :P Here it is:

Please check the method signature: https://developer.gnome.org/gtk3/stable/GtkPaned.html#gtk-paned-pack1

File: synfig-studio/src/synfigapp/actions/valuedescexport.cpp (hunk 2 above)
Comment (morevnaproject) [reply synfig-studio/src/synfigapp/actions/valuedescexport.cpp:61]: I believe this references Library Panel where all exported values stored. That way user have a hint where to look for list of exported values. ^__^

**Comment by morevnaproject** [comment]
@flurick Can you please split this PR into separate PRs?

File: synfig-studio/src/gui/docks/dock_toolbox.cpp (hunk 1 above)
Comment (ice0) [reply synfig-studio/src/gui/docks/dock_toolbox.cpp:96]: @rodolforg 
It looks like he is right. This is the method signature:
`void pack1(Widget& child, bool resize, bool shrink);`

**Comment by ice0** [comment]
@morevnaproject 
> Can you please split this PR into separate PRs?

I think it's ok to merge it as-is.

File: synfig-studio/src/gui/docks/dock_toolbox.cpp (hunk 1 above)
Comment (rodolforg) [reply synfig-studio/src/gui/docks/dock_toolbox.cpp:96]: The second argument is boolean, but he uses flags...

Comment (ice0) [reply synfig-studio/src/gui/docks/dock_toolbox.cpp:96]: @rodolforg 
This code was there before. Only the last flag has changed.

Comment (rodolforg) [reply synfig-studio/src/gui/docks/dock_toolbox.cpp:96]: It was wrong before XD

**Review by ice0 — APPROVED** [review]

**Comment by ice0** [comment]
Merged. Thank you!

**Comment by morevnaproject** [comment]
Can be backported to 1.4.3, but needs #2436,a s there is an issue with Panel Tabs (#2435). ^__^

**Pull request merged**